locales/en/home.json

```
@@ -4,5 +4,11 @@
   "categorySectionTitle": "CATEGORIES",
   "searchSectionTitle": "COBDA, PRESENTED BY BANK.",
   "searchSectionDescription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec mimagna, placerat eu semper at, pretium sit amet libero. Integerultrices tristique quam.",
-  "searchPlaceholder": "What are you looking for?"
+  "searchPlaceholder": "What are you looking for?",
+  "newReleaseSectionTitle": "NEW RELEASE",
+  "newReleaseSectionHeader": "<0>NEW<1/> RELEASES</0>",
```

Do you think we should remove this?
```suggestion
  "newReleaseSectionHeader": "<0>NEW<1/>RELEASES</0>",
```